backend/store/etcd/tessen_store.go

```diff
@@ -0,0 +1,24 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sensu/sensu-go/backend/tessen"
+)
+
+func getTessenPath() string {
+	return fmt.Sprintf("%s/tessen", EtcdRoot)
+}
+
+// CreateOrUpdateTessenConfig creates or updates the tessen configuration
+func (s *Store) CreateOrUpdateTessenConfig(ctx context.Context, config *tessen.Config) error {
+	return CreateOrUpdate(ctx, s.client, getTessenPath(), "", config)
```

Comment:
👍👍👍 for using the generic store functions haha

Reply:
The less code, the better! Thanks for providing them 😉